hs/Server.hs

```diff
@@ -58,6 +63,9 @@ runServer = do
               dir "depts" $ liftIO deptList,
               dir "timesearch" searchResponse,
               dir "calendar" $ lookCookieValue "selected-lectures" >>= calendarResponse,
+              -- dir "draw-json?gid=num" $ look "gid" >>= drawJsonResponse,
```

Comment:
The commented out areas represent my first attempt at creating the "/draw-json" response. At this point, they are unnecessary, and I will get rid of all of them.

Also, the professor wanted to test out the "Pull" functionality, because I've never used it before.

Should commented-code be avoided in the future?